### ISSUE TITLE
Tabbed Focus of Add Files and Add Folder "Buttons"

### ISSUE THREAD

**Issue by mtribone** [samvera/hyrax#1246]
### Descriptive summary

In Hyrax 2.0.0alpha, when adding files to a work via keyboard navigation the styled inputs do not receive focus when tabbing through the interface. 

### Rationale

When using the keyboard for navigation and interacting with the interface, a user is unable to a) know that they are able to interact with an element without focus and b) activate a user interface element with the keyboard. Interacting and navigating via keyboard is important to the user experience and for those who have differing abilities.

### Expected behavior

I expect to navigate to and interact with the add files and add folder "buttons" via the keyboard and that the interface have focus when tabbing through the interface.

### Actual behavior

The interface elements do not receive focus and I am unable to activate them via the keyboard. 

### Steps to reproduce the behavior

1. Create a new work or edit an existing work
1. Tab through the interface and see if the "buttons" receive focus
1. try and add files or a folder using the keyboard

### Related work

https://github.com/psu-stewardship/scholarsphere/pull/452

**Comment by adamjarling** [comment]
This also seems to be an issue on the Collections > Branding page.  Will add to the fix list for this issue.

![collections-branding](https://user-images.githubusercontent.com/3020266/43791962-e30085a8-9a3c-11e8-8e77-9237ff1046ca.png)

**Comment by no-reply** [comment]
See aecfe67, wrt reopening.

**Comment by mcdowney** [comment]
Closing as fixed in 3.0
